docs/apps/interchain-accounts/auth-modules.md

```diff
@@ -1,399 +1,21 @@
 <!--
-order: 7
+order: 3
 -->
 
 # Building an authentication module
 
-### Deprecation Notice
+Authentication modules enable application developers to perform custom logic when interacting with the Interchain Accounts controller sumbmodule's `MsgServer`. {synopsis}
 
-**This document is deprecated and will be removed in future releases**.
+The controller submodule is used for account registration and packet sending. It executes only logic required of all controllers of interchain accounts. The type of authentication used to manage the interchain accounts remains unspecified. There may exist many different types of authentication which are desirable for different use cases. Thus the purpose of the authentication module is to wrap the controller submodule with custom authentication logic.
```

Comment:
Some diagrams might be helpful? Adi has posted quite a few in [this blog post](https://medium.com/the-interchain-foundation/ibc-go-v6-changes-to-interchain-accounts-and-how-it-impacts-your-chain-806c185300d7)